index.html

```diff
@@ -0,0 +1,86 @@
+    />
+    <script
+      src="https://kit.fontawesome.com/de30ca7aea.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Velvet Video Library</title>
+  </head>
+
+  <body>
+    <header>
+      <!-- Navigation Bar -->
+      <nav class="nav-box-bar desktop-nav">
+        <div id="nav-section-bar">
+          <div id="logo-bar">
+            <a>
+              <p class="logo-text"><span>Velvet</span> Videos</p>
+            </a>
+          </div>
+        </div>
+
+        <div class="login-nav-box">
+          <div id="social-links-bar">
+            <button class="no-bg hov user-icon">
+              <i class="fa-solid fa-user"></i>
+            </button>
+            <button class="no-bg hov btn-solid-primary">Login</button>
+          </div>
+        </div>
+      </nav>
+      <!-- End of Navigation bar -->
+    </header>
+
+    <!-- Banner -->
+    <section class="banner-section">
+      <div class="banner-intro">
+        <img src="./assets/banner.jpg" />
+      </div>
+      <div class="layout-above">
+        <p>Experience the Himalayas!</p>
+        <button class="btn-solid-primary">Watch now</button>
+      </div>
+    </section>
+    <!-- End of Banner -->
+
+    <section class="must-watch-section">
+      <p class="heading-section">Must Watch Videos</p>
+      <div class="flex-center">
+        <div class="video-card">
+            <img class="video-image" src="./assets/video.jpg">
+            <p class="video-name">SEN Sinatara leaks his Team</p>
+            <p class="video-details"><span class="video-views">6K views</span> | <span class="video-hours-ago">4 hours ago</span></p>
+            <button class="btn-solid-secondary video-watch-btn">Watch now</button>
+        </div>
+        <div class="video-card">
+            <img class="video-image" src="./assets/video.jpg">
+            <p class="video-name">SEN Sinatara leaks his Team</p>
+            <p class="video-details"><span class="video-views">6K views</span> | <span class="video-hours-ago">4 hours ago</span></p>
+            <button class="btn-solid-secondary video-watch-btn">Watch now</button>
+        </div>
+        <div class="video-card">
+            <img class="video-image" src="./assets/video.jpg">
+            <p class="video-name">SEN Sinatara leaks his Team</p>
+            <p class="video-details"><span class="video-views">6K views</span> | <span class="video-hours-ago">4 hours ago</span></p>
+            <button class="btn-solid-secondary video-watch-btn">Watch now</button>
+        </div>
+        <div class="video-card">
+            <img class="video-image" src="./assets/video.jpg">
+            <p class="video-name">SEN Sinatara leaks his Team</p>
+            <p class="video-details"><span class="video-views">6K views</span> | <span class="video-hours-ago">4 hours ago</span></p>
+            <button class="btn-solid-secondary video-watch-btn">Watch now</button>
+        </div>
+        
+      </div>
+    </section>
+  </body>
+</html>

```

Comment:
look into those suggestions other than that rest of the code looks good you can proceed to merge...